components/Goals.vue

```diff
@@ -17,7 +17,6 @@
             height="110"
             src="https://www.youtube.com/embed/cCOL7MC4Pl0"
             title="YouTube video player"
-            frameborder="0"
```

Comment:
Why are we removing the `frameborder` attribute? Any reason for keeping the border?

Reply:
This is a deprecated attribute. We should control the visuals of the border using CSS instead.

Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe